src/ray/rpc/grpc_callback_client.h

```diff
@@ -0,0 +1,336 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <grpcpp/grpcpp.h>
+#include <grpcpp/impl/codegen/client_callback.h>
+
+#include <boost/asio.hpp>
+
+#include "ray/common/grpc_util.h"
+#include "ray/common/ray_config.h"
+#include "ray/common/status.h"
+#include "ray/rpc/client_call.h"
```

Comment:
What are we using from this file?

Reply:
We're only using the `ClientCallback` type definition. If we want to remove that dependency, we can duplicate that type definition in this file.